draft-packages/modal/KaizenDraft/Modal/illustrations/cautionary.icon.svg

```diff
@@ -0,0 +1,34 @@
+<svg width="154" height="154" viewBox="0 0 154 154" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
These are spot illustrations https://www.figma.com/file/GMxm8rvDCbj0Xw3TQWBZ8b/Zen-UI-Kit?node-id=1929%3A4212

They're typically hosted in [kaizen-design-system-assets](https://github.com/cultureamp/kaizen-design-system-assets/blob/master/assets/illustrations/spot/moods-cautionary.svg) with a React wrapper around them [here](https://github.com/cultureamp/kaizen-design-system/blob/master/draft-packages/illustration/KaizenDraft/Illustration/Spot.tsx)

Reply:
oh, wasn't aware that that was a thing. Will update soon